CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ### Major Changes
 
+- Removes the "Add product" section from admin orders shipments tab
+
 ### Core
 
 ### Backend
 
+- Removes the "Add product" section from admin orders shipments tab
```

Comment:
What do you think about keeping the format of the rest of the CHANGELOG? I mean, adding a reference to this same PR since the number could be useful to easily find the context of this change.

Reply:
Actually I thought about not adding this at all, as we generate the changelog from PRs before we make a release. But the PR description asked me to add changelog entries. We should remove the changelog note from the PR description IMO. We can't expect from contributors to maintain a changelog in a PR (The don't know the PR # and the release this is being merged in anyway)

Reply:
You are right, going to remove that line now